packages/core/src/shared/utilities/cliUtils.ts

```diff
@@ -285,9 +285,7 @@ async function installSsmCli(
         await new TimedProcess('pkgutil', pkgArgs).run({ spawnOptions: { cwd: tempDir } })
         await new TimedProcess('tar', tarArgs).run({ spawnOptions: { cwd: tempPath } })
 
-        fs.copySync(path.join(tempPath, 'usr', 'local', 'sessionmanagerplugin'), outDir, {
-            recursive: true,
-        })
+        await fs.copy(path.join(tempPath, 'usr', 'local', 'sessionmanagerplugin'), outDir)
```

Comment:
`fs.copy` needs a docstring, it's not obvious that it also supports folders.

Reply:
tracked as a followup, seems like there a few other methods that could use them as well in `fs`.